movie-app/src/scss/components/body/_card.scss

```diff
@@ -9,10 +9,10 @@
   align-items: flex-start;
   position: relative;
   overflow: hidden;
-  min-height: 450px;
-  min-width: 300px;
-  max-height: 450px;
-  max-width: 300px;
+  min-height: 28.125em;
+  min-width: 18.75em;
+  max-height: 28.125em;
+  max-width: 18.75em;
```

Comment:
Glad this worked out!  Be sure that this component's direct parent has a font size specified for you to base the em scaling on.

Reply:
While `em` is really nice for keeping font-size to component-size ratios, it's important to note that it inherits that scaling factor from the nearest parent with a font-size specified.  Say that the nearest two parents don't specify font-size, but the container that contains them does.  If you change the font size of that container, it will also affect the em scaling.

Reply:
Good to know. Thanks for the clarification.